web/ui/react-app/src/pages/graph/GraphHelpers.ts

```diff
@@ -4,6 +4,7 @@ import { escapeHTML } from '../../utils';
 import { Metric } from '../../types/types';
 import { GraphProps, GraphData, GraphSeries, GraphExemplar } from './Graph';
 import moment from 'moment-timezone';
+import { ListOfColors } from './ColorHelper';
```

Comment:
Nit: prefixing a list / array with `ListOf` is pretty uncommon in general, I'd probably just call the constant `colors` or `colorPool` and the file `Colors.ts`.

Reply:
Yup, Colorpool is better, will update it

Reply:
Thanks! For the file, I still think `Colors.ts` is better, since `Helper` would imply to me some kind of active code and not just a constant list.

Reply:
(or `ColorPool.ts`)

Reply:
I just named it colorHelper as even in the future, we can keep all our color-related codes in one place. Because having a file with only one list does not make that much sense.

Reply:
But let's keep it ColorPool.ts as of now, maybe in future if we want a whole new helper of colors we can shift names at that time.